controllers/controller.js

```diff
@@ -7,3 +7,11 @@ exports.getCategories = (req, res, next) => {
     })
     .catch(next);
 };
+
+exports.getReviews = (req, res, next) => {

```

Comment:
Think about separating out controllers for categories and reviews e.g. categories.controllers.js, reviews.controllers.js